<launch>

  <node pkg="cb_tracker"
        type="cb_detector.py" name="cb_detector" >
    <param name="corners_x" type="int" value="6" />
    <param name="corners_y" type="int" value="8" />
    <param name="spacing_x" type="double" value="0.036" />
    <param name="spacing_y" type="double" value="0.036" />
    <param name="baseframe" type="string" value="kingfisher/axis" />
    <remap from="image_stream" to="/axis/image_raw" />
    <remap from="camera_info" to="/axis/camera_info" />
  </node>

</launch>
